metaflow/plugins/kfp/kfp.py

```diff
@@ -713,8 +713,6 @@ def _set_container_labels(
                 )
             container_op.add_pod_annotation(annotation_name, annotation_value)
 
-        # TODO(talebz): A Metaflow plugin framework to customize tags, labels, etc.
-        container_op.add_pod_label("aip.zillowgroup.net/kfp-pod-default", "true")
         # tags.ledger.zgtools.net/* pod labels required for the ZGCP Costs Ledger
         container_op.add_pod_label("tags.ledger.zgtools.net/ai-flow-name", self.name)
         container_op.add_pod_label("tags.ledger.zgtools.net/ai-step-name", node.name)
```

Comment:
One thing to think about is whether we want to add in these labels & annotations to all the steps too?

I think having the flow/step/run_id on the `s3_sensor` and `exit-handler` (and any other step for that matter) seems to make sense to me. I wonder if we can just call this method from `pipeline_transform`?

Reply:
Thank you, `s3_sensor` could take a while.  
Created [AIP-6887](https://zbrt.atl.zillow.net/browse/AIP-6887) WFSDK: add annotations to all pods (+exit_handler, s3_sensor)
I wanted to keep AIP-6884 as minimal as possible to fix only the bug at hand. 
thank you!